dev/tasks/gandiva-jars/build-java.sh

```diff
@@ -23,9 +23,33 @@ CPP_BUILD_DIR=$TRAVIS_BUILD_DIR/dist/
 
 pushd java
   if [ $TRAVIS_OS_NAME == "linux" ]; then
-    ldd $CPP_BUILD_DIR/libgandiva_jni.so
+    SO_DEP=ldd
```

Comment:
should we do this config based..so that it does not affect the nightly ones that the arrow community has..

Reply:
Added the check inside an if condition to see if environment variable CHECK_SHARED_DEPENDENCIES is set. The variable can be set in travis repository settings.